modules/core/src/test/java/org/apache/ignite/testsuites/IgnitePdsMvccTestSuite4.java

```diff
@@ -62,6 +63,7 @@
         ignoredTests.add(PageLockTrackerManagerTest.class);
         ignoredTests.add(SharedPageLockTrackerTest.class);
         ignoredTests.add(ToFileDumpProcessorTest.class);
+        ignoredTests.add(ToStringDumpHelperTest.class);
```

Comment:
Ok, I see that you added it to ignored tests somewhere, but where is the actual suit that will run it?